pyuvsim/source.py

```diff
@@ -35,6 +37,29 @@ class SkyModel(object):
 
     Used to calculate local coherency matrix for source brightness in AltAz
     frame at a specified time.
+
```

Comment:
I'm not sure how, but it looks like the docstring for the __init__ function got moved here.

Reply:
I moved it on purpose. The numpy style says that `__init__` docstrings should go on the object. This is important because methods that start with underscores don't get built automatically using sphinx.

Reply:
Oh I see now. Sorry, I misread the diff and thought it had been moved to the `_skymodel_basesize` function instead.